projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -317,7 +325,7 @@ export function NewsletterAccessDocumentSettings( {
 												help={ __( 'Control how this newsletter is viewed.', 'jetpack' ) }
 											/>
 											<NewsletterAccessRadioButtons
-												onChange={ setPostMeta }
+												onChange={ setPostMetaAndClose( onClose ) }
```

Comment:
I would  have lazily done 
```suggestion
												onChange={ (meta) => { setPostMeta( meta ); onClose(); }  }
```

I really like the generator. I miss them :)